src/config/authd-config.c

```diff
@@ -224,6 +231,11 @@ int Read_Authd(const OS_XML *xml, XML_NODE node, void *d1, __attribute__((unused
 
         config->force_options.enabled = legacy_force_insert;
     }
+    if (legacy_force_time > 0) {
+        mdebug1("Setting <force><enabled> tag to %s to comply with the legacy <%s> option found.",
+                legacy_force_time ? "'yes'" : "'no'", xml_force_time);
+        config->force_options.enabled = legacy_force_time;
```

Comment:
`legacy_force_time` cannot be written in `config->force_options.enabled`. It has to be stored in `config->force_options.disconnected_time` and you have to add `disconnected_time_enabled = 1`.